sample-xml/app/src/main/java/co/nimblehq/sample/xml/extension/NavArgsExt.kt

```diff
@@ -0,0 +1,10 @@
+package co.nimblehq.sample.xml.extension
+
+import androidx.annotation.MainThread
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavArgs
+import androidx.navigation.fragment.navArgs
+
+@MainThread
+inline fun <reified Args : NavArgs> Fragment.provideNavArgs(): Lazy<Args> =
+    OverridableLazy(navArgs())
```

Comment:
I think we should bring this good update to `template-xml`.

Reply:
Added in this https://github.com/nimblehq/android-templates/pull/450/commits/da634200a4bfe703fc2e947435c7f2ebf0c76c8d ✅